src/components/Board/Board.messages.js

```diff
@@ -1,6 +1,14 @@
 import { defineMessages } from 'react-intl';
 
 export default defineMessages({
+  undo: {
+    id: 'cboard.components.Board.undo',
```

Comment:
When refactoring to `UndoButton`, should update this string:
` id: 'cboard.components.UndoButton.undo'`